File: src/main/java/com/zxx/demorepository/DemorepositoryApplication.java
package com.zxx.demorepository;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.AdviceMode;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableAutoConfiguration
@EnableScheduling
@EnableAsync
@MapperScan("com.zxx.demorepository.test.mapper")
public class DemorepositoryApplication {

    public static void main(String[] args) {
        SpringApplication.run(DemorepositoryApplication.class, args);
    }
}

File: src/main/java/com/zxx/demorepository/DemorepositoryApplication.kt
package com.zxx.demorepository

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.scheduling.annotation.EnableAsync
import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.*

@SpringBootApplication(scanBasePackages = ["com.zxx.demorepository"])
@EnableScheduling
@EnableAsync
@MapperScan("com.zxx.demorepository.test.mapper")
class DemorepositoryApplication

fun main(args: Array<String>) {
    runApplication<DemorepositoryApplication>(*args)
}
